### DISCUSSION TITLE
Can i access files directly with verdaccio url (like CDN).

### DISCUSSION THREAD

**Discussion by jaisonka91** [verdaccio/verdaccio#3074] [Q&A] [answered]
Im using verdaccio from last 1 year. Now i have a new requirement like i want to access 1 css file and js file directly from the register server. Can i do it with verdaccio?

**Comment by juanpicado** [comment, marked as answer]
I am not sure if I follow, you want to use a CDN to serve only the assets for the UI? Maybe you can elaborate

**Reply by jaisonka91** [reply]
There is one css file and main bunldle js, i want it to be accessed by url. Currently im installing the package and importing it.

**Reply by juanpicado** [reply]
If you want files in CDN, you can put them there. I still don't fully get your question, anyway, Have you looked into `VERDACCIO_PUBLIC_URL`?

https://verdaccio.org/docs/configuration#url-prefix

**Reply by jaisonka91** [reply]
I think this may help me, let me try with this. Thanks a lot :)

**Reply by jaisonka91** [reply]
Do we have to restart verdaccio after adding public VERDACCIO_PUBLIC_URL? @juanpicado if needed can you give me the command for restarting.

**Reply by juanpicado** [reply]
yes, any configuration you change verdaccio must be restarted, remember is just a Node.js application.

I'm not sure how are you running verdaccio., npm, or docker, in any case, you can do it yourself, either stop the docker image or just kill the verdaccio process in npm and restart.
